examples/datadog/agent_confd.yaml

```diff
@@ -0,0 +1,25 @@
+# Datadog Agent with additional check configurations
+
+datadog:
+  apiKey: <DATADOG_API_KEY>
+  appKey: <DATADOG_APP_KEY>
+  clusterName: <CLUSTER_NAME>
+  tags: []
+  # Each key becomes a file in /conf.d and overwrite default settings
```

Comment:
It may be worthwhile to include a snippet for `datadog.ignoreAutoConf`, as while these samples are for the `kubelet` and `kube_proxy` integration if a user creates a `redisdb.yaml` technically this doesn't overwrite the `redisdb.d/auto_conf.yaml`. 

Even leaving blank but exposing that might be good here:
```yaml
  ## ref: https://docs.datadoghq.com/agent/faq/auto_conf/
  ignoreAutoConfig: []
  #  - redisdb
  #  - kubernetes_state
```

Reply:
Makes sense. Include the option but remove `- redisdb` and `- kubernetes_state` to keep it simple. Instead, added a comment to explain this option (copied from public doc).
And document link is updated to point more specific section.